addon/validations/result.js

```diff
@@ -64,7 +64,7 @@ var ValidationsObject = Ember.Object.extend({
       let attrMeta = model.get('constructor.attributes').get(attribute);
       if (attrMeta) {
         let defaultValue = attrMeta.options.defaultValue;
-        if (defaultValue) {
+        if (!Ember.isNone(defaultValue)) {
```

Comment:
Can you change this from `!Ember.isNone` to `!isNone` since its already defined as a const up top